examples/randomwalks/randomwalks.py

```diff
@@ -0,0 +1,105 @@
+import networkx as nx
```

Comment:
Do we want to add `networkx` as a dependency? I believe this is what you're referring to in [this](https://github.com/CarperAI/trlx/pull/119#issuecomment-1331418733) comment? I don't see it in the `setup.cfg` 😅 - We can also just point users to `pip install networkx` in the `randomwalks/README.md` which might be better. What do you think?

Reply:
I was referring to `from examples.randomwalks import generate_random_walks`, but I also added networkx to make it frictionless